blinkpy/blinkpy.py

```diff
@@ -114,6 +120,15 @@ def start(self):
             if self.auth.no_prompt:
                 return True
             self.setup_prompt_2fa()
+
+        if not self.last_refresh:
+            # Initialize last_refresh to be just before the refresh delay period.
+            self.last_refresh = int(time.time() - self.refresh_rate * 1.05)
```

Comment:
This `interval = self.blink.last_refresh - self.motion_interval * 60` (in `check_new_videos()`) throws an exception on the first `refresh()` call.  Then a second call to `refresh()` is needed, but is prevented if it is within the refresh delay period.

`last_refresh` should be initialized.  It could be at the startup time, but initializing it to one delay period prior allows `check_new_videos()` to capture any clips triggered between the startup time and the first refresh call.

By not initializing `last_refresh`, the user must wait minimum one delay period (if they call refresh immediately) or maximum two delay periods (if they don't call refresh immediately).  Initializing it this way will get all the new clips after startup is finished.

It is a simple change that doesn't break anything, and has huge benefit.